editor/project_converter_3_to_4.cpp

```diff
@@ -2241,24 +2241,25 @@ void ProjectConverter3To4::process_gdscript_line(String &line, const RegExContai
 
 	// rotating = true  ->   ignore_rotation = false # reversed "rotating" for Camera2D
 	if (contains_function_call(line, "rotating")) {
-		int start = line.find("rotating");
+		String function_name = "rotating";
+		int start = line.find(function_name);
 		bool foundNextEqual = false;
-		String line_to_check = line.substr(start + String("rotating").length());
+		String line_to_check = line.substr(start + function_name.length());
 		String assigned_value;
 		for (int current_index = 0; line_to_check.length() > current_index; current_index++) {
 			char32_t chr = line_to_check.get(current_index);
 			if (chr == '\t' || chr == ' ') {
 				continue;
 			} else if (chr == '=') {
 				foundNextEqual = true;
-				assigned_value = line.right(current_index).strip_edges();
+				assigned_value = line.trim_prefix(line.left(function_name.length() + start + current_index + 1)).strip_edges();
```

Comment:
That seems a bit convoluted, isn't the problem just that `right` was used instead of `substr`?

Reply:
Nevermind, `substr` can indeed be used to simplify the code. That's what happens when you code at 4 am. 😅